### PR TITLE
clippy fixes

### PR DESCRIPTION
**Pull request by matthiaskrgr**

### CODE REVIEW

**Comment by ytmimi** [comment]
These changes look good to me

File: src/expr.rs [hunk 1]
```diff
@@ -2003,9 +2003,8 @@ fn choose_rhs<R: Rewrite>(
     has_rhs_comment: bool,
 ) -> Option<String> {
     match orig_rhs {
-        Some(ref new_str) if new_str.is_empty() => {
-            return Some(String::new());
-        }
+        Some(ref new_str) if new_str.is_empty() => Some(String::new()),
+
```
Comment (calebcartwright) [inline src/expr.rs:2007]: Could you remove the extra line added here? Almost hate to ask but we apparently maintain line breaks between arms, and I don't think this one was added intentionally

Comment (ytmimi) [reply src/expr.rs:2007]: sorry for not catching that on my initial review. I don't know how I overlooked the extra line.

Comment (calebcartwright) [reply src/expr.rs]: No worries it was a bit of a nit anyway, thanks for reviewing!

**Pull request merged**